package.json

```diff
@@ -25,7 +25,9 @@
     "test": "yarn workspace client run test"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^3.3.0"
+    "@aave/protocol-v2": "^1.0.1",
+    "@openzeppelin/contracts": "^3.3.0",
+    "@truffle/hdwallet-provider": "^1.2.1"
```

Comment:
Would this be better as a devDependency?

Reply:
yup, makes sense to me!